cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -183,9 +184,9 @@ public class AppLifecycleHttpHandler extends AbstractAppFabricHttpHandler {
   public BodyConsumer create(HttpRequest request, HttpResponder responder,
                              @PathParam("namespace-id") final String namespaceId,
                              @PathParam("app-id") final String appId)
-    throws BadRequestException, NamespaceNotFoundException, AccessException {
+          throws BadRequestException, NamespaceNotFoundException, AccessException {
```

Comment:
Why change the indentation?

Reply:
My IDE settings were off. Fixing it in the next revision